home.css

```diff
@@ -0,0 +1,613 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'lato', sans-serif;
+  min-height: 100vh;
+  background-color: #e5e5e527;
+}
+
+.social-media,
+.navbar {
+  display: none;
+}
+
+.box {
+  background: url('./images/desktop.png') no-repeat;
+  background-size: cover;
+  margin-bottom: 0;
+  padding-bottom: 50px;
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  position: relative;
+  margin-top: 8%;
+  padding-left: 5%;
+}
+
+.hamburger img {
+  width: 60%;
+  height: 70%;
+  margin-bottom: 80%;
+}
+
+.hello {
+  position: relative;
+  color: #f35826;
+  font-size: 200%;
+  margin-bottom: 0%;
+  padding-left: 5%;
+}
+
+.heading {
+  position: relative;
+  color: #f35826;
+  font-size: 400%;
+  margin-top: 0%;
+  font-weight: bold;
+  padding-left: 5%;
+}
+
+/* intro section */
+.intro {
+  border: 5px #fff solid;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-right: 15%;
+  margin-top: 5%;
+  font-weight: normal;
+  opacity: 0.7;
+}
+
+.due-date {
+  position: relative;
+  font-weight: bold;
+  color: #585858;
+  font-size: 200%;
+  margin-left: 5%;
+  margin-bottom: 1%;
+  margin-top: 5%;
+}
+
+.venue {
+  color: #222121;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-bottom: 5%;
+  opacity: 0.7;
+}
+
+/* Main program section */
+.program {
+  background: url('./images/main-program.jpg') no-repeat 0, 0 #0a0a0a;
+  background-size: cover;
+  margin-top: 0%;
+}
```

Comment:
- [ ] Following the project requirements, the pages use the exact same colors, for text, backgrounds, borders, etc, as the original design.
- Basic color swatch: #272A31, #EC5242, #D3D3D3

- Your design (The background color is black):
<img width="842" alt="image" src="https://user-images.githubusercontent.com/77190203/176998633-1a2edd42-79c1-4725-857c-46387eee7bad.png">

- Original design:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/77190203/176998660-19b9ba9e-560a-422b-a0be-fa2626bfd7e0.png">